client/extensions/woocommerce/state/ui/payments/methods/test/selectors.js

```diff
@@ -87,6 +87,23 @@ describe( 'selectors', () => {
 			] );
 		} );
 
+		test( 'should apply the "edits" to title to title attribute and settings value', () => {
```

Comment:
I think that I am reading this correctly and the two `to title` make sense. Just checking.

Reply:
Yeah that wording is a bit funky, I might re-write that sentence. I try to avoid the "to to" thing.